corehq/apps/accounting/admin.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.contrib import admin
+
+from .models import DomainUserHistory
+
+
+class DomainUserHistoryAdmin(admin.ModelAdmin):
+    model = DomainUserHistory
+    list_display = ('domain', 'record_date', 'num_users')
+    list_filter = ('domain', 'record_date')
+
+
+admin.site.register(DomainUserHistory, DomainUserHistoryAdmin)
```

Comment:
fwiw I prefer the style of using decorators, since in a large file it's still easy to tell which model admins the model goes with:
https://github.com/dimagi/commcare-hq/pull/24311/commits/1e98fadb4f277e487ae83a4e33431e4be11f5eac

Reply:
Definitely better, updated.